src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import "../styles/App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Home from "../containers/Home";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" exact element={<Home />} />
```

Comment:
define constants folder in the project and create links.js file 

inside the constants folder, there should be 2 files 
- index.js (to export all the files of this folder from index.js file `export * from './links.js'`)
- links.js (inside links.js you will create `export const LINKS={home:{path:'/'}}`)

Reply:
Done